.gitignore

```
@@ -1,3 +1,4 @@
 submit_files/*
 input_files/*
-output_files/*
\ No newline at end of file
+output_files/*
+src/__pycache__/*
```

Files should have a trailing newline. It is easy to forget to do this
```suggestion
src/__pycache__/*

```